docs/rl-algorithms/td3.md

```diff
@@ -0,0 +1,109 @@
+# Twin Delayed Deep Deterministic Policy Gradient (TD3)
+
+
+## Overview
+
+TD3 is a popular DRL algorithm for continuous control. It extends DQN to work with the continuous action space by introducing a deterministirc actor that directly outputs continuous actions. DDPG also combines techniques from DQN such as the replay buffer and target network.
+
+
+Original paper: 
+
+* [Addressing Function Approximation Error in Actor-Critic Methods](https://arxiv.org/abs/1802.09477)
+
+Reference resources:
+
+* :material-github: [sfujim/TD3](https://github.com/sfujim/TD3)
+* [Twin Delayed DDPG | Spinning Up in Deep RL](https://spinningup.openai.com/en/latest/algorithms/td3.html)
+
+## Implemented Variants
+
+
+| Variants Implemented      | Description |
+| ----------- | ----------- |
+| :material-github: [`td3_continuous_action.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/td3_continuous_action.py), :material-file-document: [docs](/rl-algorithms/td3/#td3_continuous_actionpy) | For continuous action space |
+
+
+Below are our single-file implementations of PPO:
```

Comment:
Given that there is only a single implementaion: "is" for now ?

Reply:
It's fine as a template.